Makefile

```diff
@@ -2,6 +2,8 @@
 SHELL := /bin/bash -o pipefail -o errexit -o nounset
 
 addlicense=go run github.com/google/addlicense@v1.0.0
+version?=0.0.3-$(shell git rev-parse --short HEAD)
```

Comment:
I dont like the manual 0.0.3 for the same reasons of having the VERSION file, but at least this is used only for make related automation (dev stuff) + it is very easy to define a different version if you want.

Reply:
this should be read from VERSION file... automation scripts should never have hardcoded stuff...

Reply:
we could also automate a way to get the latest stable version of the project like this:

```
go mod download -json github.com/mineiros-io/terramate@latest
{
        "Path": "github.com/mineiros-io/terramate",
        "Version": "v0.0.3",
        "Info": "/home/katz/go/pkg/mod/cache/download/github.com/mineiros-io/terramate/@v/v0.0.3.info",
        "GoMod": "/home/katz/go/pkg/mod/cache/download/github.com/mineiros-io/terramate/@v/v0.0.3.mod",
        "Zip": "/home/katz/go/pkg/mod/cache/download/github.com/mineiros-io/terramate/@v/v0.0.3.zip",
        "Dir": "/home/katz/go/pkg/mod/github.com/mineiros-io/terramate@v0.0.3",
        "Sum": "h1:vzsfu8FX6IwF6D4MPeUJUdpqPlmMerPJ68Nt+EtfIsM=",
        "GoModSum": "h1:M7Gn5dOycbD/vShHG+xpBvImUirliXeaGVhjEncMIXg="
}
```

It gets the latest tagged version, so it would automatically assume the most updated latest stable version.